api/app/views/spree/api/shipments/big.v1.rabl

```diff
@@ -41,6 +41,13 @@ child order: :order do
 
   child payments: :payments do
     attributes :id, :amount, :display_amount, :state
+    child source: :source do |s|
+      if s.is_a?(Spree::CreditCard)
```

Comment:
@gmacdougall  Sounds good, added another commit to address this and hopefully make adding more attributes in the future a little easier.